scripts/build_static_site.sh

```diff
@@ -103,27 +97,19 @@ build() {
     sourcecred_data="$(mktemp -d --suffix ".sourcecred-data")"
     export SOURCECRED_DIRECTORY="${sourcecred_data}"
 
-    if [ "${#projects[@]}" -ne 0 ]; then
-        local weightsStr=""
-        if [ -n "${weights}" ]; then
-            weightsStr="--weights ${weights}"
-        fi
-        for project in "${projects[@]}"; do
-            NODE_PATH="/code/node_modules${NODE_PATH:+:${NODE_PATH}}" \
-                node /code/bin/sourcecred.js load "${project}" $weightsStr
-        done
+    # Load project files (required)
+    if [ "${#project_files[@]}" -eq 0 ]; then
+        die 'project-file is required.'
     fi
 
-    if [ "${#project_files[@]}" -ne 0 ]; then
-        local weightsStr=""
-        if [ -n "${weights}" ]; then
-            weightsStr="--weights ${weights}"
-        fi
-        for project_file in "${project_files[@]}"; do
-            NODE_PATH="/code/node_modules${NODE_PATH:+:${NODE_PATH}}" \
-                node /code/bin/sourcecred.js load --project "${project_file}" $weightsStr
-        done
+    local weightsStr=""
+    if [ -n "${weights}" ]; then
+        weightsStr="--weights ${weights}"
     fi
+    for project_file in "${project_files[@]}"; do
+        NODE_PATH="/code/node_modules${NODE_PATH:+:${NODE_PATH}}" \
+            node /code/bin/sourcecred.js load --project "${project_file}" $weightsStr
```

Comment:
@Beanow see here, this is the confusing bit. The --project variable requires the project file, not the project. It threw me through a loop :)